subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1602,9 +1602,16 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_cmd_ap,
 		  "Disable Access Point mode",
```

Comment:
Minor nit, as this commit is also for shell, perhaps set the commit subject to `wifi: shell: ...` like in the other commit.